units/user-config.target

```diff
@@ -11,3 +11,9 @@ After=user-cloudinit@var-lib-coreos\x2dinstall-user_data.path
 
 Requires=user-cloudinit-proc-cmdline.service
 After=user-cloudinit-proc-cmdline.service
+
+Requires=user-config-vmware-tools.service
```

Comment:
I don't think we want to support this use case just yet. Optimistically running all of the various agents is a bit expensive. If we did want to support this, we would want a better system for starting these services; maybe a series of OEM targets (e.g. vmware.target) that started the appropriate services, mounts, timers, etc..